infrastructure/frontend/frontend.template.yml

```diff
@@ -142,6 +142,8 @@ Resources:
               Value: !Ref ContainerPort
             - Name: SHOW_TEST_BANNER
               Value: !Ref ShowTestBanner
+            - Name: GOOGLE_SHEETS_INTEGRATION_ENABLED
```

Comment:
@kalpaitch Wonder if I can get a sense check from you on this commit?

Context: The change we're adding involves some new  ENV Vars.
They fall into a few categories:
- x2 feature flags (should differ between environments but are not secret)
- x1 new stub toggle (similar story, but also same as what we do with sheets + service now)
- x3 configuration variables (I've hard coded these, they should be the same everywhere, there's no integration Jira for now, but flexible for future use)
- x2 secrets (these could vary by env, but are secret and so I think we need them in SSM)

I wouldn't mind a sense check that I'm using cloudformation correctly here.
I've also not created any SSM params in any env yet.

Would appreciate a quick chat about how best to do that, avoiding clickops wherever possible for the audit trail?

Reply:
Hardcoded values could be ssm param. This is a good practice.

Reply:
That's a good set of rules of thumb, thanks @kalpaitch and @rafal-surowiec-co

Reply:
Have moved the static values into a config file `config/jiraConfig.ts`, so I'm gonna mark this one as resolved unless there are any objections